ros-ocp/templates/deployment-redis.yaml

```diff
@@ -50,7 +76,11 @@ spec:
           readinessProbe:
             exec:
               command:
+                {{- if $isOpenShift }}
```

Comment:
can you extract this into a helper function and capture the value at the beginning of the template instead of calling this logic again and again?

Reply:
wdyt?